services/graphite/errors.go

```diff
@@ -0,0 +1,12 @@
+package graphite
+
+import "fmt"
+
+type UnsupportedValueError struct {
```

Comment:
We use a naming convention with an `Err` prefix.  So this should be `ErrUnsupportedValue`.